staging/src/k8s.io/kubectl/pkg/util/prune/prune.go

```diff
@@ -22,8 +22,27 @@ import (
 
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/klog/v2"
 )
 
+// default allowlist without non-namespaced resources
+var defaultPruneResources = []Resource{
```

Comment:
Suggestion: make this `defaultNamespacedPruneResources` and add `defaultNonNamespacedPruneResources` with the other two instead of hardcoding them inline.